vagen/envs/tos/managers/cognitive_map_manager.py

```
@@ -0,0 +1,1516 @@
+        for fb_turn in fb_turn_logs:
+            cm_log = ((fb_turn.get('false_belief_log') or {}).get('cogmap_log') or {})
+            for m in (cm_log.get('unchanged_retention_per_object') or {}).values():
+                if not isinstance(m, dict):
+                    continue
+                if isinstance(m.get('pos'), (int, float)):
+                    pos_vals.append(float(m['pos']))
+                if isinstance(m.get('facing'), (int, float)):
+                    facing_vals.append(float(m['facing']))
+        if not pos_vals and not facing_vals:
+            return {}
+        avg = lambda v: sum(v) / len(v) if v else None
```

Ruff (0.15.2)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `avg` as a `def`

(E731)